ui/components/ui/icon/help-outline-icon.component.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function HelpOutLineIcon({
```

Comment:
```suggestion
export default function HelpOutlineIcon({
```

Reply:
fixed in 0fd7dc7f22cf45741ed4b85432f2dd1a2ea4ae83

Reply:
- Verified